bsdate_test.go

```diff
@@ -157,6 +158,9 @@ func TestConversionToGregorian(t *testing.T) {
 var impossibleToConvertToGregorianDates = [] string {
 	"1970-01-01",
 	"1970-08-29",
+	"2040-13-20",
```

Comment:
this will already fail in `New(bsDay, bsMonth, bsYear)` because its an invalid BS date. In this test valid BS dates are tested that still cannot be converted to Gregorian, the test description is actually not so good

Reply:
fixed